packages/@adobe/spectrum-css-temp/components/textfield/index.css

```diff
@@ -116,6 +117,14 @@ governing permissions and limitations under the License.
        font-style: normal;
     }
   }
+  /* placeholder gets clipped for synthetic italics, we rely on font-synthesis once
+   * chrome supports it https://bugs.chromium.org/p/chromium/issues/detail?id=509989
+   */
+  &:lang(ar), &:lang(ja), &:lang(zh), &:lang(ko) {
+    &::placeholder {
+      font-style: normal;
+    }
+  }
```

Comment:
I noticed that chrome still has some clipping in the story (built locally):
![image](https://user-images.githubusercontent.com/9661127/105910421-c73ce500-5fdd-11eb-9a62-20590c194de0.png)
Was there supposed to be a chrome specific fix?

Reply:
this is the chrome specific fix, but yours looks different than mine was, let me make sure there wasn't some issue with caching

Reply:
This is what it looks like for me with the last successful build from this branch
![Screen Shot 2021-01-26 at 2 37 39 PM](https://user-images.githubusercontent.com/698229/105915030-138b2380-5fe4-11eb-98eb-410088c0473a.png)